packages/context/src/inject.ts

```diff
@@ -124,7 +124,7 @@ export function inject(
   }
   return function markParameterOrPropertyAsInjected(
     target: Object,
-    member: string,
+    member?: string,
```

Comment:
This change is needed to allow manual invocation of the decorator - I am surprised TypeScript did not catch that.

Example usage that does not work without my patch:

```ts
inject(`repositories.${entityClass.name}Repository`)(
    controllerClass,
    undefined,
    0,
  );
```

Reply:
See https://github.com/strongloop/loopback-next/pull/3634